### PR TITLE
feat: upload new apk version [DHIS2-14962]

### PR DESCRIPTION
**Pull request by Sharmyn28**
**Implements** [DHIS2-14962](https://dhis2.atlassian.net/browse/DHIS2-14962)

---

### Key features

1. Update the namespace and keys to use in the datastore (query and mutation). _The following PRs will have the creation of Datastore and the initial setup_.
2. Update the form and fields required to upload new APK
3. Create validators for the version and URL field
4. Update the table with the APK version list
5. Update the list of Android versions

---

### Screenshots

_table with APK versions_
image

_upload new version_
image

_validation version field_
image

_update the latest version_
image

[DHIS2-14962]: https://dhis2.atlassian.net/browse/DHIS2-14962?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/UploadApk/UploadApk.js [hunk 1]
```diff
@@ -1,21 +1,155 @@
...
+                                            name="androidOSVersion.recommended"
+                                            label={i18n.t(
+                                                'Recommended Android version'
+                                            )}
+                                            placeholder={i18n.t(
+                                                'Select a version'
+                                            )}
+                                            component={SingleSelectFieldFF}
+                                            className={styles.field}
+                                            options={androidOSVersion}
```
Comment (simonadomnisoru) [inline src/pages/UploadApk/UploadApk.js:153]: Hey, 
As far as I understand the recommended version should be equal to or higher than the minimum version. Does it make sense to only show equal or higher options in this dropdown?
Currently, it is possible to create an APK with a recommended version smaller than the minimum version. Is this configuration useful to the users or is it a bug?

Screenshot 2023-04-05 at 17 06 43

Thanks!

File: src/pages/UploadApk/UploadApk.js [hunk 2]
```diff
@@ -1,21 +1,155 @@
...
+                                {({ handleSubmit, valid, submitting }) => (
+                                    <form onSubmit={handleSubmit}>
+                                        <ReactFinalForm.Field
+                                            required
+                                            name="version"
+                                            label={i18n.t('Version')}
+                                            placeholder={i18n.t('e.g. 1.0.0')}
+                                            component={InputFieldFF}
+                                            className={styles.field}
+                                            validate={composeValidators(
```
Comment (simonadomnisoru) [inline src/pages/UploadApk/UploadApk.js:111]: Should the app version name be unique? Currently, it is possible to add a new version with the same name as an existing one. 

Screenshot 2023-04-05 at 17 01 22

Thanks!

Comment (Sharmyn28) [reply src/pages/UploadApk/UploadApk.js:111]: Thanks for the advice. I'm adding a validation so it's only possible to add a unique version higher than the latest.

Comment (Sharmyn28) [reply src/pages/UploadApk/UploadApk.js:111]: image

image

**Review by simonadomnisoru — APPROVED** [review]

**Pull request merged**